justfile

```
@@ -366,6 +366,8 @@ rctemplate := '''
 container_registry=""
 # Azure resource group/ resource name. Resource group will be created.
 azure_resource_group=""
+# Platform to deploy on
+default_platform=""
```

```suggestion
default_platform="AKS-CLH-SNP"
```

Maybe we can set a sane default?